pkg/KubeArmorOperator/config/crd/bases/operator.kubearmor.com_kubearmorconfigs.yaml

```diff
@@ -0,0 +1,82 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: kubearmorconfigs.operator.kubearmor.com
+spec:
+  group: operator.kubearmor.com
+  names:
+    kind: KubeArmorConfig
```

Comment:
no i don't think so, we support multiple deployment options i.e. helm and operator we are managing these configurations using ConfigMap in general and watching for the changes in KubeArmor. this CR will be the only source of the configuration when operator is being used to deploy the kubearmor. 

but yes i agree, currently it's possible to  have different configurations with this CR and the Configmap.

Reply:
@rksharma95 Just to make sure we are on the same page, will `karmor install` install the operator or kubearmor directly?

Reply:
IMO we shouldn't be asking users to install kubearmor directly using Helm charts because there is no way to handle environment specific volumes etc automatically in Helm. Helm does support pre-install hooks however it is triggered only after kubernetes templates are generated from charts.
So, the users will have to specify it by overriding the `--environment.name` value so that proper container runtime etc maybe be used. It is very confusing and error prone.

Reply:
imo `karmor install` should install KubeArmor directly, as operator requires user to deply operator CRs to deploy the KubeArmor. wdyt ?

Reply:
I think karmor could still create the KubeArmorConfig CRs, if that's the need.
Let's get opinion on this from others as well?